apollo-router/src/router_factory.rs

```diff
@@ -127,12 +127,11 @@ async fn create_plugins(
     schema: &Schema,
 ) -> Result<Vec<(String, Box<dyn DynPlugin>)>, BoxError> {
     // List of mandatory plugins. Ordering is important!!
-    let mut mandatory_plugins = vec!["experimental.include_subgraph_errors", "apollo.csrf"];
-
-    // Telemetry is *only* mandatory if the global subscriber is set
-    if crate::subscriber::is_global_subscriber_set() {
-        mandatory_plugins.insert(0, "apollo.telemetry");
-    }
+    let mandatory_plugins = vec![
+        "experimental.include_subgraph_errors",
+        "apollo.csrf",
+        "apollo.telemetry",
```

Comment:
now the telemetry plugin must be always set, or we won't have any logs after configuration is loaded

Reply:
if people want to provide their own subscriber, how should we proceed? If the configuration file does not show anything related to telemetry, we might still want to see logs in stdout

Reply:
fixed in [fc4dd14](https://github.com/apollographql/router/pull/1463/commits/fc4dd149e2760a39916f19b4fee37d18de351d71), now we can add a subscriber explicitely